monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md

```diff
@@ -0,0 +1,129 @@
+# MONERO
+
+Monero est une cryptomonnaie sécurisée,
+confidentielle et intraçable disponible pour
+tous
+
+# QUELQUES FAITS
+
+Les utilisateurs peuvent la dépenser en
+toute sécurité, personne ne peut voir leurs
+soldes ou suivre leurs activités.
+
+Révisé le 14/11/2018 - [Version Imprimable](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
```

Comment:
The link is incorrect

[Version Imprimable](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)

Correct link:
http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_fr_PrinterFriendly.pdf